sql/pypi_client.go

```diff
@@ -14,6 +14,8 @@ type pypiVersionResponse struct {
 
 const astroSQLCliProjectURL = "https://pypi.org/pypi/astro-sql-cli/json"
 
+var getPypiVersion = GetPypiVersion
+
 func GetPypiVersion(projectURL string) (string, error) {
```

Comment:
```suggestion
var getPypiVersion = func (projectURL string) (string, error) {
```
nit: cleaner way to do the same thing

Reply:
We need the function to reassign it to the variable in the tests after having monkey-patched, because the successive tests use the same monkey-patched version otherwise. https://github.com/astronomer/astro-cli/pull/849/files#diff-0d310e499467646455c981ef796fc947f43f23205c90e7806505c0541fe295c5R77

Reply:
syntactically both ways represent & achieve the same thing, but I am fine with whatever you might be comfortable with